app/products/calls/actions/calls.ts

```diff
@@ -399,8 +399,8 @@ export const getEndCallMessage = async (serverUrl: string, channelId: string, cu
 
     msg = intl.formatMessage({
         id: 'mobile.calls_end_msg_channel',
-        defaultMessage: 'Are you sure you want to end a call with {numSessions} participants in {displayName}?',
-    }, {numSessions, displayName: channel.displayName});
+        defaultMessage: 'Are you sure you want to end a call with {numParticipants} participants in {displayName}?',
```

Comment:
Changing the variable name would break languages until they are updated, we sure we want to do that?

Reply:
Same answer.

All the languages I check are using `numParticipants`, so this string should be broken for all the languages as of today. This would actually fix that.